docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -1266,7 +1266,13 @@ hidden by default since most users will be uninterested in changing them.
             a reference to objects it manipulates.  Most useful for
             non-reference-counted implementations of Python, like PyPy
             (where it is enabled by default).
-            
+
+        ``CYTHON_AVOID_THREAD_UNSAFE_BORROWED_REFS``
+            Avoid using APIs that return "borrowed references" and instead use
+            the equivalent APIs that return "strong references". Most useful for
+            the free-threaded build of CPython, where incrementing the reference
+            count of borrowed references might introduce thread safety issues.
```

Comment:
```suggestion
            Avoid using APIs that return unsafe "borrowed references" and instead use
            the equivalent APIs that return "strong references". Most useful for
            the free-threaded build of CPython, where incrementing the reference
            count of borrowed references to items in mutable containers might 
            introduce thread safety issues. Borrowed refernces to items in immutable
            containers are still allowed with this setting.
```

Reply:
```suggestion
            Avoid using APIs that return unsafe "borrowed references" and instead use
            the equivalent APIs that return "strong references". Most useful for
            the free-threaded build of CPython, where incrementing the reference
            count of borrowed references to items in mutable containers might 
            introduce thread safety issues. Borrowed references to items in immutable
            containers are still allowed with this setting.
```

Reply:
There was a typo in the last sentence, that's why I did a new suggestion. Thanks for the improvement!